### PR TITLE
Address #230: update first chapters

### PR DESCRIPTION
**Pull request by zolkis**
Also, fix ReSpec errors (on streams).

***
<a href="https://pr-preview.s3.amazonaws.com/zolkis/wot-scripting-api/pull/234.html" title="Last updated on Aug 19, 2020, 12:03 PM UTC (1ec0a9e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/234/16a28ee...zolkis:1ec0a9e.html" title="Last updated on Aug 19, 2020, 12:03 PM UTC (1ec0a9e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,17 +208,31 @@
...
+          <li>Remove a <a>Property</a> definition from the <a>Thing</a>.</li>
+          <li>Add an <a>Action</a> definition to the <a>Thing</a>.</li>
+          <li>Remove an <a>Action</a> definition from the <a>Thing</a>.</li>
+          <li>Add a WoT <a>Event</a> definition to the <a>Thing</a>.</li>
+          <li>Remove a WoT <a>Event</a> definition from the <a>Thing</a>.</li>
+        </ul>
+        <p class="ednote">
+          After evaluating dynamic modifications to <a>Thing Descriptions</a>
+          through several versions of this API, the editors concluded that the
+          simplest way to represent these use cases it to take an existing
```
Comment (relu91) [inline index.html:231]: Missing *is*: *...these use cases it to...* ->  *...these use cases it IS to...*

Comment (zolkis) [reply index.html]: Thanks for spotting. I corrected and will squash the commit in the first one before merging.

File: index.html [hunk 2]
```diff
@@ -116,10 +116,14 @@
...
     </p>
     <p>
-      Scripting is an optional "convenience" building block in WoT and it is typically used in gateways that are able to run a <a>WoT Runtime</a> and <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager">script management</a>, providing a convenient way to extend WoT support to new types of endpoints and implement WoT applications such as <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/thing-directory">Thing Directory</a>.
+      The main <a>Web of Things</a> (WoT) concepts are described in the <a href="https://www.w3.org/TR/2019/CR-wot-architecture-20190516/">WoT Architecture</a> document.
+    </p>
+    <p>
+      Scripting is an optional building block in WoT and it is typically used in gateways or browsers that are able to run a <a>WoT Runtime</a> and
+      <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager">script management</a>, providing a convenient way to extend WoT support to new types of endpoints and implement WoT applications such as <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/thing-directory">Thing Directory</a>.
```
Comment (danielpeintner) [inline index.html:126]: w.r.t. https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager
I am not 100% sure how we deal with it but https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager uses a *very* out-dated TD form.
Shall I try to propose a PR with an updated form. Is there more we should do?

w.r.t. https://github.com/w3c/wot-scripting-api/tree/master/applications/thing-directory
Also outdated code snippets... mhhh

File: index.html [hunk 3]
```diff
@@ -128,7 +132,7 @@
...
     </p>
     <p class="ednote">
-      This specification is implemented at least by the <a href="http://www.thingweb.io/">Thingweb</a> project also known as <a href="https://github.com/eclipse/thingweb.node-wot">node-wot</a>, which is considered the reference open source implementation at the moment. Check its <a href="https://github.com/eclipse/thingweb.node-wot"> source code</a>, including <a href="https://github.com/eclipse/thingweb.node-wot">examples</a>. Other, closed source implementations have been made by WG member companies and tested against <a href="https://github.com/eclipse/thingweb.node-wot">node-wot</a> in plug-fests.
+      This specification is implemented at least by the <a href="http://www.thingweb.io/">Thingweb</a> project also known as <a href="https://github.com/eclipse/thingweb.node-wot">node-wot</a>, which is considered the reference open source implementation at the moment. Check its <a href="https://github.com/eclipse/thingweb.node-wot"> source code</a>, including <a href="https://github.com/eclipse/thingweb.node-wot/examples">examples</a>.
```
Comment (danielpeintner) [inline index.html:135]: I think @egekorkan  was also mentioning that we should add Eclipse.. maybe something like 

<a href="http://www.thingweb.io/">Eclipse Thingweb</a>

File: index.html [hunk 4]
```diff
@@ -224,6 +243,10 @@
           <li>
             to invoke an <a>Action</a>: take the parameters from the request, execute the defined action, and return the result;
           </li>
+          <li>to observe a <a>Property</a>;</li>
```
Comment (danielpeintner) [inline index.html:246]: the listed items look fine. maybe we should group them. First properties, then action and lastly events...

File: index.html [hunk 5]
```diff
@@ -116,10 +116,14 @@
...
+      The Web of Things is made of entities (<a>Thing</a>s) that can describe their capabilities in a machine-interpretable <a>Thing Description</a> (TD) and expose these capabilities through the <a>WoT Interface</a>, that is, network interactions modeled as <a>Properties</a> (for reading and writing values), <a>Action</a>s (to execute remote procedures with or without return values) and <a>Event</a>s (for signaling notifications).
     </p>
     <p>
-      Scripting is an optional "convenience" building block in WoT and it is typically used in gateways that are able to run a <a>WoT Runtime</a> and <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/script-manager">script management</a>, providing a convenient way to extend WoT support to new types of endpoints and implement WoT applications such as <a href="https://github.com/w3c/wot-scripting-api/tree/master/applications/thing-directory">Thing Directory</a>.
+      The main <a>Web of Things</a> (WoT) concepts are described in the <a href="https://www.w3.org/TR/2019/CR-wot-architecture-20190516/">WoT Architecture</a> document.
```
[the remark this answers is not in the document]
Comment (zolkis) [reply index.html:122]: We should maybe just use biblio entries instead of direct links.

File: index.html (hunk 2 above)
Comment (zolkis) [reply index.html:126]: Yes, we discussed in yesterday's call that we need to update the auxiliary documents, including the script manager (or remove the reference from here).

Same for Thing Directory (which got  a new definition in the current charter).

**Review by danielpeintner — APPROVED** [review]

**Pull request merged**